Source/WebKit/UIProcess/API/wpe/WPEView.cpp

```diff
@@ -319,6 +319,9 @@ View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseC
 
 View::~View()
 {
+    wpe_view_backend_set_backend_client(m_backend, nullptr, nullptr);
+    wpe_view_backend_set_input_client(m_backend, nullptr, nullptr);
```

Comment:
I would also call `wpe_view_backend_set_fullscreen_client(m_backend, nullptr, nullptr);` here, but it triggers an [assert in WPE](https://github.com/WebPlatformForEmbedded/libwpe/blob/d7c669ca6f5ec0d544c264016d270669b336c931/src/view-backend.c#L106) which needs to be addressed first:

```
    [pid=939644][err] MiniBrowser: /root/webkit/WebKitBuild/DependenciesWPE/Source/libwpe-1.14.1/src/view-backend.c:103: wpe_view_backend_set_fullscreen_client: Assertion `!backend->fullscreen_client' failed.
```

Reply:
Heh, you anticipated my review feedback. I think we should just remove that assert from libwpe. Can you report a libwpe bug for this please? Bonus points for a pull request.

Reply:
I uploaded [a PR](https://github.com/WebPlatformForEmbedded/libwpe/pull/129) to libwpe, waiting for review.